### PR TITLE
Provides a better error message on 404s

### LINKED ISSUE #62: Provide good error message when an @import 404s
This is related to the changes in steal 1.7.0, but might have to be handled differently here since dependencies are not part of the steal graph.

### PR DESCRIPTION
**Pull request by matthewp**
This provides a better error message on 404s and links to the
stealjs.com documentation about the topic.

Closes #62 

screen shot 2018-03-20 at 4 31 49 pm

### CODE REVIEW

File: less.js [hunk 1]
```diff
@@ -97,7 +98,27 @@ exports.translate = function(load) {
...
 				lessEngine.render(load.source, renderOptions)
 			);
-			return p.then(done);
+			return p.then(done).then(null, function(err){
+				// 404.
+				//!steal-remove-start
+				if(err.type === "File" && /404/.test(err.message)) {
+					if(loader._addSourceInfoToError) {
+						var fn = err.filename.split("/").pop();
+						var msg = "The stylesheet [" + fn + "] was able to fetch a dependency.\n" +
```
Comment (chasenlehara) [inline less.js]: Should “was able” be “wasn’t able”?

Comment (matthewp) [reply less.js]: yep, nice catch

Comment (chasenlehara) [reply less.js]: Yay for including screenshots in your PR (also yay for this PR in general)!

**Comment by matthewp** [comment]
screen shot 2018-03-20 at 4 53 43 pm

**Pull request merged**